Lib/test/test_xmlrpc.py

```diff
@@ -343,6 +343,78 @@ def run_server():
             self.assertEqual(p.method(), 5)
             self.assertEqual(p.method(), 5)
 
+    def test_simple_xml_rpc_dispatcher_resolves_registered_func_and_does_not_chain_excs_when_calling_it(self):
+        exp_params = 1, 2, 3
+        dispatched = False
+
+        class DispatchExc(Exception):
+            """Raised inside the dispatched func"""
+
+        def dispatched_func(*params):
+            nonlocal dispatched
+            dispatched = True
+            self.assertEqual(params, exp_params)
+            raise DispatchExc()
+
+        dispatcher = xmlrpc.server.SimpleXMLRPCDispatcher()
+        dispatcher.register_function(dispatched_func)
+        try:
+            dispatcher._dispatch(method='dispatched_func', params=exp_params)
+        except DispatchExc as e:
+            self.assertFalse(e.__cause__)
+            self.assertFalse(e.__context__)
+        self.assertTrue(dispatched)
+
+    def test_simple_xml_rpc_dispatcher_resolves_instance_func_and_does_not_chain_excs_when_calling_it(self):
+        exp_params = 1, 2, 3
+        dispatched = False
+
+        class DispatchExc(Exception):
+            """Raised inside the dispatched func"""
+
+        class DispatchedClass:
+            @staticmethod
```

Comment:
No other reason in particular other than PyCharm suggesting to make it static as it did not depend on `self`. But I will get rid of the static declaration.